plugin.yaml

```diff
@@ -1,7 +1,7 @@
 plugins:
   gcp_plugin:
     executor: central_deployment_agent
-    source: https://github.com/cloudify-cosmo/cloudify-gcp-plugin/archive/1.1.0.zip
+    source: https://github.com/cloudify-cosmo/cloudify-gcp-plugin/archive/add-support-for-kubernetes-engine.zip
     package_name: cloudify-gcp-plugin
     package_version: '1.1.0'
```

Comment:
Can we change this to `1.2.0.dev0` for now?

Reply:
yeah sure

Reply:
Done :)